lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -854,6 +990,12 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
   late InputConfiguration inputConfiguration;
   EditingState? lastEditingState;
 
+  TextEditingDeltaState? _editingDelta;
+  TextEditingDeltaState get editingDelta {
```

Comment:
I was confused at first by the fact that some TextEditingDeltaStates are named `delta` and some are `deltaState`.  What do you think of normalizing them all to `deltaState`?  So here `editingDelta` would become `editingDeltaState`.

Reply:
That makes sense to me. I'll do that .